### ISSUE TITLE
Update `peerDependencies` to include React 18

### ISSUE THREAD

**Issue by frankalbenesius** [splitio/react-client#93]
```zsh
❯ npm i
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: @splitsoftware/splitio-react@1.4.0
npm ERR! Found: react@18.0.0
npm ERR! node_modules/react
npm ERR!   react@"18.0.0" from the root project
npm ERR!   peer react@"^16.11.0 || ^17 || ^18" from @auth0/auth0-react@1.10.0
npm ERR!   node_modules/@auth0/auth0-react
npm ERR!     @auth0/auth0-react@"1.10.0" from the root project
npm ERR!   13 more (@emotion/react, @testing-library/react, react-dom, ...)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.3.0 || ^17.0.0" from @splitsoftware/splitio-react@1.4.0
npm ERR! node_modules/@splitsoftware/splitio-react
npm ERR!   @splitsoftware/splitio-react@"1.4.0" from the root project
```

can `peerDependencies` be updated to include "^18.0.0"?
```json
  "peerDependencies": {
    "react": "^16.3.0 || ^17.0.0"
  },
```

**Comment by chillaq** [comment]
Thanks @frankalbenesius , checking with the team.

**Comment by frankalbenesius** [comment]
> Thanks @frankalbenesius , checking with the team.

Hi @chillaq . Any word on this?

**Comment by wuweiweiwu** [comment]
@chillaq bump!

**Comment by chillaq** [comment]
Nothing yet @frankalbenesius , I pinged the team.

**Comment by chillaq** [comment]
Hi @wuweiweiwu , we released React SDK 1.4.1 that fixes this issue.

**Comment by frankalbenesius** [comment]
thank you 😄
